### DISCUSSION TITLE
A thing i made like conway's game of life

### DISCUSSION THREAD

**Discussion by loafhouse** [TodePond/CellPond#332] [Show and tell]
[life.json](https://github.com/TodePond/CellPond/files/13707458/life.json)

**Comment by TodePond** [comment]
amazing!!!! 
<img width="1138" alt="image" src="https://github.com/TodePond/CellPond/assets/15892272/cabf5d66-931a-4178-934d-183f8e0f3b0b">

**Reply by loafhouse** [reply]
holy shit! i never expected you to type this! thanks!

**Reply by TodePond** [reply]
why not? :)

**Reply by raisedeyebrowemoji** [reply]
This is the joy of being in a small youtuber's community
